svf/lib/WPA/Andersen.cpp

```diff
@@ -157,6 +159,30 @@ void AndersenBase::analyze()
         finalize();
 }
 
+
+void AndersenBase::analyzeAndWrite(const std::string& filename)
```

Comment:
analyzeAndWrite => SolveAndWritePtsToFile


SolveAndWritePtsToFile(filename){

initialize();
this->writeObjVarToFile(filename);
solveConstraints();
this->writeToFile(filename);
finalize();

}